api/scpca_portal/test/test_loader.py

```diff
@@ -0,0 +1,407 @@
+                project_zip,
+            )
+
+            # Check if correct libraries were added in
+            expected_libraries = {"SCPCL999990", "SCPCL999997"}
+            self.assertCorrectLibraries(project_zip, expected_libraries)
+
+            # Check that correct files were added in
+            # There are 14 files:
+            # ├── README.md
+            # ├── SCPCS999990
+            # │   ├── SCPCL999990_celltype-report.html
+            # │   ├── SCPCL999990_filtered.rds
+            # │   ├── SCPCL999990_processed.rds
+            # │   ├── SCPCL999990_qc.html
+            # │   └── SCPCL999990_unfiltered.rds
+            # ├── SCPCS999997
+            # │   ├── SCPCL999997_celltype-report.html
+            # │   ├── SCPCL999997_filtered.rds
+            # │   ├── SCPCL999997_processed.rds
+            # │   ├── SCPCL999997_qc.html
+            # │   └── SCPCL999997_unfiltered.rds
+            # ├── bulk_metadata.tsv
+            # ├── bulk_quant.tsv
+            # └── single_cell_metadata.tsv
+            expected_file_list = [
+                "README.md",
+                "SCPCS999990/SCPCL999990_celltype-report.html",
+                "SCPCS999990/SCPCL999990_filtered.rds",
+                "SCPCS999990/SCPCL999990_processed.rds",
+                "SCPCS999990/SCPCL999990_qc.html",
+                "SCPCS999990/SCPCL999990_unfiltered.rds",
+                "SCPCS999997/SCPCL999997_celltype-report.html",
+                "SCPCS999997/SCPCL999997_filtered.rds",
+                "SCPCS999997/SCPCL999997_processed.rds",
+                "SCPCS999997/SCPCL999997_qc.html",
+                "SCPCS999997/SCPCL999997_unfiltered.rds",
+                "SCPCP999990_bulk_metadata.tsv",
+                "SCPCP999990_bulk_quant.tsv",
+                "single_cell_metadata.tsv",
+            ]
+            result_file_list = project_zip.namelist()
+            self.assertEqual(set(expected_file_list), set(result_file_list))
+
+        # Check computed file attributes
+        computed_file = project.computed_files.filter(
+            **self.get_computed_files_query_params_from_download_config(download_config)
+        ).first()
+
+        self.assertEqual(
+            computed_file.format, ComputedFile.OutputFileFormats.SINGLE_CELL_EXPERIMENT
+        )
+        self.assertFalse(computed_file.includes_merged)
+        self.assertFalse(computed_file.metadata_only)
+        self.assertEqual(computed_file.s3_bucket, settings.AWS_S3_OUTPUT_BUCKET_NAME)
+        self.assertEqual(computed_file.s3_key, output_file_name)
+        self.assertEqual(computed_file.size_in_bytes, 9078)
+        self.assertEqual(computed_file.workflow_version, "development")
+        self.assertTrue(computed_file.includes_celltype_report)
+        self.assertTrue(computed_file.has_bulk_rna_seq)
+        self.assertFalse(computed_file.has_cite_seq_data)
+        self.assertFalse(computed_file.has_multiplexed_data)
```

Comment:
Convert computed file expected attributes to a dictionary and then iterate over the dict with an 
```python
for key, value in expected_computed_file_attributes:
    assertEqual(getattr(computed_file, key), value)
```